_data/deployments.yml

```diff
@@ -12,6 +12,7 @@ Deis: "/user/deployment/deis/"
 Engine Yard: "/user/deployment/engineyard/"
 ExoScale: "/user/deployment/exoscale/"
 GitHub Releases: "/user/deployment/releases/"
+GitHub Pages: "/user/deployment/pages"
```

Comment:
Good point on the order! I'll leave the quotes just to be consistent with the rest of the existing document though